optimize/backend/src/main/java/io/camunda/optimize/service/status/StatusCheckingServiceES.java

```diff
@@ -7,55 +7,41 @@
  */
 package io.camunda.optimize.service.status;
 
-import io.camunda.optimize.rest.engine.EngineContextFactory;
 import io.camunda.optimize.service.db.es.OptimizeElasticsearchClient;
 import io.camunda.optimize.service.db.schema.OptimizeIndexNameService;
-import io.camunda.optimize.service.importing.ImportSchedulerManagerService;
-import io.camunda.optimize.service.util.configuration.ConfigurationService;
 import io.camunda.optimize.service.util.configuration.condition.ElasticSearchCondition;
 import jakarta.ws.rs.core.Response;
-import lombok.extern.slf4j.Slf4j;
 import org.elasticsearch.action.admin.cluster.health.ClusterHealthRequest;
 import org.elasticsearch.action.admin.cluster.health.ClusterHealthResponse;
 import org.elasticsearch.cluster.health.ClusterHealthStatus;
 import org.springframework.context.annotation.Conditional;
 import org.springframework.stereotype.Component;
 
 @Component
-@Slf4j
 @Conditional(ElasticSearchCondition.class)
 public class StatusCheckingServiceES extends StatusCheckingService {
 
   private final OptimizeElasticsearchClient esClient;
 
   public StatusCheckingServiceES(
       final OptimizeElasticsearchClient esClient,
-      final ConfigurationService configurationService,
-      final EngineContextFactory engineContextFactory,
-      final ImportSchedulerManagerService importSchedulerManagerService,
       final OptimizeIndexNameService optimizeIndexNameService) {
-    super(
-        configurationService,
-        engineContextFactory,
-        importSchedulerManagerService,
-        optimizeIndexNameService);
+    super(optimizeIndexNameService);
     this.esClient = esClient;
   }
 
   @Override
   public boolean isConnectedToDatabase() {
-    boolean isConnected = false;
     try {
       ClusterHealthRequest request =
           new ClusterHealthRequest(optimizeIndexNameService.getIndexPrefix() + "*");
       final ClusterHealthResponse healthResponse = esClient.getClusterHealth(request);
 
-      isConnected =
-          healthResponse.status().getStatus() == Response.Status.OK.getStatusCode()
-              && healthResponse.getStatus() != ClusterHealthStatus.RED;
+      return healthResponse.status().getStatus() == Response.Status.OK.getStatusCode()
+          && healthResponse.getStatus() != ClusterHealthStatus.RED;
     } catch (Exception ignored) {
       // do nothing
     }
-    return isConnected;
+    return false;
```

Comment:
should this be in the catch block?

Reply:
Adjusted this to match the OS one, can do it this way around too though